systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -685,36 +763,28 @@ void testDynamicallySetClusterOperatorLoggingLevels() {
             });
     }
 
-    /**
-     * @description This test case verifies dynamic and non-dynamic changes in KafkaConnect logging level.
-     *
-     * @steps
-     *  1. - Deploy Kafka cluster and KafkaConnect cluster, the latter with Log level Off.
-     *  2. - Deploy all additional resources, scraper Pod and network policies.
-     *  3. - Verify that no logs are present in KafkaConnect Pods.
-     *  4. - Set inline log level to Debug in KafkaConnect CustomResource.
-     *     - log4j.properties file for given cluster has log level Debug, and pods provide logs on respective level.
-     *  5. - Change inline log level from Debug to Info in KafkaConnect CustomResource.
-     *     - log4j.properties file for given cluster has log level Info, and pods provide logs on respective level.
-     *  6. - Create ConfigMap with necessary data for external logging and modify KafkaConnect CustomResource to use external logging setting log level Off.
-     *     - log4j.properties file for given cluster has log level Off, and pods provide no more logs.
-     *  7. - Disable the use of connector resources via annotations and verify KafkaConnect pod rolls.
-     *     - Verify that KafkaConnect deployment rolls
-     *  8. - Enable DEBUG logging via inline configuration
-     *     - Verify log lines contain DEBUG level messages after configuration changes.
-     *  9. - Check the propagation of logging level change to DEBUG under the condition where connector resources are not used.
-     *     - Validate that DEBUG logs are present and correct according to the log4j pattern.
-     * @usecase
-     *  - logging
-     *  - logging-change
-     *  - kafka-connect
-     *  - dynamic-configuration
-     *  - rolling-update
-     */
     @ParallelNamespaceTest
     @Tag(ROLLING_UPDATE)
     @Tag(CONNECT)
     @Tag(CONNECT_COMPONENTS)
+    @TestDoc(
+        description = @Desc("This test case verifies dynamic and non-dynamic changes in KafkaConnect logging level."),
+        steps = {
+            @Step(value = "Deploy Kafka cluster and KafkaConnect cluster, the latter with Log level Off.", expected = "Clusters are deployed with KafkaConnect log level set to Off."),
+            @Step(value = "Deploy all additional resources, scraper Pod and network policies.", expected = "Additional resources, scraper Pod and network policies are deployed."),
```

Comment:
I would also say why the NP are created -> in order to gather the stuff from the Scraper Pod.